lib/cloud_foundry.rb

```diff
@@ -11,6 +11,15 @@ def self.vcap_data
     end
   end
 
+  def self.app_url
+    if self.is_environment?
+      # (arbitrarily) use the shortest assigned route, if there are multiple
```

Comment:
Well, when we do [zero-downtime deploys](https://github.com/18F/cf-blue-green), the "green" applications (and routes) are named `<appname>-B`. I've also manually set up routes like `<appname>-test`. Strong feelings?